src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,106 @@
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
+        case Status.COMPLETED:
+          return completed;
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
+        default:
+          return true;
+      }
+    });
+  });
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+  return (
+    <>
+      {sortedTodos.length === 0
+        ? (
+          <Error>
+            There are no todos matching current filter criteria
+          </Error>
+        )
+        : (
+          <table className="table is-narrow is-fullwidth">
+            <thead>
+              <tr>
+                <th>#</th>
+                <th>
+                  <span className="icon">
+                    <i className="fas fa-check" />
+                  </span>
+                </th>
+                <th>Title</th>
+                <th> </th>
+              </tr>
+            </thead>
+            <tbody>
+              {sortedTodos.map((todo) => (
+                <tr
+                  data-cy="todo"
+                  className=""
+                  key={todo.id}
+                >
+                  <td className="is-vcentered">{todo.id}</td>
+                  <td className="is-vcentered">
+                    {todo.completed && (
+                      <span className="icon" data-cy="iconCompleted">
+                        <i className="fas fa-check" />
+                      </span>
+                    )}
+                  </td>
+                  <td className="is-vcentered is-expanded">
+                    <p className={classNames({
+                      'has-text-danger': !todo.completed,
+                      'has-text-success': todo.completed,
+                    })}
+                    >
+                      {todo.title}
+                    </p>
+                  </td>
+                  <td className="has-text-right is-vcentered">
+                    <button
+                      data-cy="selectButton"
+                      className="button"
+                      type="button"
+                      onClick={() => (
+                        dispatch(currTodoActions.setTodo(todo))
+                      )}
+                    >
+                      <span className="icon">
+                        <i className={classNames(
+                          'far',
+                          {
+                            'fa-eye': selectedTodo?.id !== todo.id,
+                            'fa-eye-slash': selectedTodo?.id === todo.id,
```

Comment:
You can move this condition into variable to not repeat yourself
```suggestion
                            'fa-eye': !isCurrentTodo,
                            'fa-eye-slash': isCurrentTodo,
```